tracee-ebpf/tracee/argprinters.go

```diff
@@ -51,130 +51,212 @@ func (t *Tracee) parseArgs(event *external.Event) error {
 		if int32(event.EventID) == CapCapableEventID {
 			if capArg := getEventArg(event, "cap"); capArg != nil {
 				if capability, isInt32 := capArg.Value.(int32); isInt32 {
-					capArg.Value = helpers.ParseCapability(capability)
 					capArg.Type = "string"
+					capArg.Value = "" // arg has to always exist (even if errors below)
+					capabilityFlagArgument, err := helpers.ParseCapability(uint64(capability))
+					if err != nil {
+						return err
+					}
+					capArg.Value = capabilityFlagArgument.String()
 				}
 			}
 		}
 		if int32(event.EventID) == SecurityFileOpenEventID {
 			if flagsArg := getEventArg(event, "flags"); flagsArg != nil {
 				if flags, isInt32 := flagsArg.Value.(int32); isInt32 {
-					flagsArg.Value = helpers.ParseOpenFlags(uint32(flags))
 					flagsArg.Type = "string"
+					flagsArg.Value = "" // arg has to always exist (even if errors below)
+					openFlagArgument, err := helpers.ParseOpenFlagArgument(uint64(flags))
+					if err != nil {
+						return err
+					}
+					flagsArg.Value = openFlagArgument.String()
 				}
 			}
 		}
 	case MmapEventID, MprotectEventID, PkeyMprotectEventID:
 		if protArg := getEventArg(event, "prot"); protArg != nil {
 			if prot, isInt32 := protArg.Value.(int32); isInt32 {
-				protArg.Value = helpers.ParseMemProt(uint32(prot))
+				mmapProtArgument := helpers.ParseMmapProt(uint64(prot))
+				protArg.Value = mmapProtArgument.String()
 				protArg.Type = "string"
 			}
 		}
 	case PtraceEventID:
 		if reqArg := getEventArg(event, "request"); reqArg != nil {
 			if req, isInt64 := reqArg.Value.(int64); isInt64 {
-				reqArg.Value = helpers.ParsePtraceRequest(req)
 				reqArg.Type = "string"
+				reqArg.Value = "" // arg has to always exist (even if errors below)
+				ptraceRequestArgument, err := helpers.ParsePtraceRequestArgument(uint64(req))
+				if err == nil {
+					reqArg.Value = ptraceRequestArgument.String()
+				} // TODO: https://github.com/aquasecurity/libbpfgo/issues/108
```

Comment:
Opened to fix this: https://github.com/aquasecurity/libbpfgo/pull/110

This PR along with https://github.com/aquasecurity/libbpfgo/pull/109 can get merged and we can do another quick release of libbpfgo, wdyt?

Reply:
Done. Let's hold another release for now, let me try to adequate this PR to something acceptable to @yanivagman first. I'll change the error handling and address his concerns.